terraform/releases/impl/promote-release.tf

```diff
@@ -197,7 +197,17 @@ resource "aws_iam_role_policy" "promote_release" {
           "${aws_s3_bucket.static.arn}/doc/*",
           "${aws_s3_bucket.static.arn}/dist",
           "${aws_s3_bucket.static.arn}/dist/*",
-
+        ]
+      },
+      {
+        Sid    = "BucketsReadDelete"
+        Effect = "Allow"
+        Action = [
+          "s3:GetObjectAcl",
+          "s3:GetObject",
+          "s3:DeleteObject",
```

Comment:
Hm, I actually am not sure we need delete permissions on the CI artifacts bucket -- I think we should only read from that bucket. Are you aware of some deletion in our scripts?